config/game_options.txt

```diff
@@ -309,7 +309,7 @@ SEC_START_BRIG
 
 ### ROLE-SPECIFIC TRAITOR ITEMS ###
 ## Uncomment to restrict role-specific traitor items to what traitors start the round as, not as their ID.
-#ROUNDSTART_ROLE_UPLINK_ITEMS
+ROUNDSTART_ROLE_UPLINK_ITEMS
```

Comment:
But what is its default value? Because this won't be in the config unless it's put in.